### PR TITLE
OSDOCS#11053: Removing incorrect name attribute for rosa edit mpool

### PR DESCRIPTION
**Pull request by laubai**
Version(s): 4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-11053

Link to docs preview:
- https://78480--ocpdocs-pr.netlify.app/openshift-rosa/latest/cli_reference/rosa_cli/rosa-manage-objects-cli.html
- https://78480--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_cluster_admin/rosa-configuring-pid-limits.html
- https://78480--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/removing-custom-config-from-machinepool.adoc [hunk 1]
```diff
@@ -21,13 +21,13 @@ To remove all `KubeletConfig` objects from the machine pool, set an empty value
 +
 [source,terminal]
 ----
-$ rosa edit machinepool -c <cluster_name> --kubeletconfigs="" --name <machinepool_name>
+$ rosa edit machinepool -c <cluster_name> <machinepool_name> --kubeletconfigs=""
```
Comment (robpblake) [inline modules/removing-custom-config-from-machinepool.adoc:24]: @laubai I'd suggest keeping the `<machinepool_name>` at the end of the command for consistency:

```
rosa edit machinepool -c <cluster_name> --kubeletconfigs="" <machinepool_name>
```

File: modules/setting-higher-pid-limit-on-machinepool.adoc [hunk 2]
```diff
@@ -42,28 +42,28 @@ $ rosa create kubeletconfig -c my-cluster --name=set-high-pids --pod-pids-limit=
...
 ----
-$ rosa create machinepool -c <cluster_name> --kubelet-configs=<kubeletconfig_name> --name <machinepool_name>
+$ rosa create machinepool -c <cluster_name> --name <machinepool_name> --kubelet-configs=<kubeletconfig_name>
 ----
 ** For an existing machine pool:
 +
 [source,terminal]
 ----
-$ rosa edit machinepool -c <cluster_name> --kubelet-configs=<kubeletconfig_name> --name <machinepool_name>
+$ rosa edit machinepool -c <cluster_name> <machinepool_name> --kubelet-configs=<kubeletconfig_name>
```
Comment (robpblake) [inline modules/setting-higher-pid-limit-on-machinepool.adoc:51]: @laubai Same here, I'd suggest we make the `<machinepool_name>` the last part of the command

File: modules/setting-higher-pid-limit-on-machinepool.adoc [hunk 3]
```diff
@@ -42,28 +42,28 @@ $ rosa create kubeletconfig -c my-cluster --name=set-high-pids --pod-pids-limit=
...
 ----
 --
 +
 For example, the following command associates the `set-high-pids` `KubeletConfig` object with the `high-pid-pool` machine pool in the `my-cluster` cluster:
 +
 [source,terminal]
 ----
-$ rosa edit machinepool -c my-cluster --kubelet-configs=set-high-pids --name=high-pid-pool
+$ rosa edit machinepool -c my-cluster high-pid-pool --kubelet-configs=set-high-pids
```
Comment (robpblake) [inline modules/setting-higher-pid-limit-on-machinepool.adoc:59]: @laubai Same comment as above.

Comment (laubai) [reply modules/setting-higher-pid-limit-on-machinepool.adoc:59]: Good plan, thanks for the catch!

**Review by xenolinux — COMMENTED** [review]
LGTM

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Review by sheriff-rh — APPROVED** [review]
Changes look good to me, merging to 4.15+.

**Comment by sheriff-rh** [comment]
Looks like this one will also need a manual cherrypick @laubai

**Pull request merged**